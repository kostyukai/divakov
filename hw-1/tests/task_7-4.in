124.2142421
