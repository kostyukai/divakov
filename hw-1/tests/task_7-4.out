01000000000001011111100001101101101100010010001111111001110110101111100111011010
